pkgs/development/libraries/qt-5/qtModule.nix

```diff
@@ -22,7 +23,12 @@ mkDerivation (args // {
   inherit pname version src;
   patches = (args.patches or []) ++ (patches.${pname} or []);
 
-  nativeBuildInputs = (args.nativeBuildInputs or []) ++ [ perl qmake ];
+  nativeBuildInputs =
+    (args.nativeBuildInputs or []) ++ [
+      perl qmake
+    ] ++ lib.optionals (stdenv.buildPlatform != stdenv.hostPlatform) [
+      pkgsHostTarget.qt5.qtbase.dev
```

Comment:
why, the packages there shouldn't be executable?

Reply:
Normally no, except it's QT, and QT has both `qmake`-which-runs-on-the-buildPlatform as well as `libQtBase`-which-is-built-for-the-hostPlatform mashed into the same package.